password.py

```diff
@@ -1,293 +1,361 @@
 # Password Creator/Manager
-# Notes: Three Dictionaries!
-# - Make sure to use json.loads b4 every edit so it adds it to the dictionary 
-# example:
-# (acc1 : {accinfo {site : pass}},
-#  acc2 : {accinfo {site : pass}})
 
 import random
 import re
 import json
-import pathlib
-import shane_char
+import sys
+import os
+import get_char
 
 class account:
-    def __init__(self):
-        self.acc = {}
-        self.acc_info = {}
+    user_acc = {}
+    user_info = {}
+    user_pass = {}
+    filename = ""
 
-# Creates an account
-def account_create():
-    acc_data = account()
+    def __init__(self, filename):
+        self.filename = filename
 
-    print("-"*17, "\nAccount Creation!"), print("-"*17)
+    # Creates an account
+    def acc_create(self):
+        print("-"*17, "\nAccount Creation!"), print("-"*17)
 
-    while True:
-        username = input("Enter a Username:\n")
-        if len(username) < 5:
-            print("Username has to be 5 characters or more.")
-            continue
-
-        acc_data.acc_info["Username"] = username
-        break
+        while True:
+            username = input("Enter a Username:\n")
+            if len(username) < 5:
+                print("Username has to be 5 characters or more.")
+                continue
 
-    create_password(acc_data.acc_info)
+            self.user_info["Username"] = username
+            break
 
-    get_security_questions(acc_data.acc_info)
+        self.user_acc[username] = self.user_info
+        self.user_info["Password"] = self.create_pass()
```

Comment:
I think what you mean to do here is define user_info first and then pass that into `user_acc[username]`. The way you're doing it now is kinda backwards and might not work unless python is passing user.info as a pointer(we can talk more about pointers later). 

You also don't have to define user_info as a class variable, you should probably just use `self.user_acc[username][someparamhere]` if you want to access the inner dictionary. For example(in sudo code):

```suggestion
        info = {
            password: self.create_pass()
        }
        self.user_acc[username] = info
```